### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rushikeshacharya**
Made changes in README

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Make some edits on your own copy that fixes the issues.
+
+:white_check_mark:
+Create a **new branch and then commit**.
+
+:white_check_mark:
+Create a pull request that will be merged to the main repo.
+(**Remember to mention which issue you're fixing**)
+
+:white_check_mark:Your code will be reviewed and approved.
```
Comment (balajipachai) [inline docs/contributing.md]: It should be, **The code will be reviewed and approved**

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+:white_check_mark:If you have questions on any issue, kindly use the comment.
```
Comment (balajipachai) [inline docs/contributing.md]: The statement has to be:
If you have questions on any issue, kindly use the comment section.

Comment (rushikeshacharya) [reply docs/contributing.md]: Okay.

File: docs/contributing.md (hunk 1 above)
Comment (rushikeshacharya) [reply docs/contributing.md]: Okay, Thanks!

**Comment by balajipachai** [comment]
Hi, @rushikeshacharya Thank You, very much for this help, looking forward to much more contributions, appreciated !!!!

**Pull request merged**